src/sql/workbench/contrib/dashboard/browser/contents/widgetContent.component.ts

```diff
@@ -76,7 +76,7 @@ export class WidgetContent extends AngularDisposable implements AfterViewInit {
 	@Input() private widgets: WidgetConfig[];
 	@Input() private originalConfig: WidgetConfig[];
 	@Input() private context: string;
-	@Input() private scrollContent = true;
+	@Input() private scrollContent = false;
```

Comment:
why this?

Reply:
I want to use the outer container (dashboardPage.component.ts) to control the overflow behavior.